kong/cmd/utils/prefix_handler.lua

```diff
@@ -103,6 +103,33 @@ local function gen_default_ssl_cert(kong_config, target)
   return true
 end
 
+
+local function gen_trusted_certs_file(kong_config)
+  -- create SSL folder
+  local ssl_path = pl_path.join(kong_config.prefix, "ssl")
```

Comment:
I don't think we necessarily have to create this directory. We can save the bundle to `$prefix/.ca_bundle` or something similar, like how we saved `.kong_env`. This way it is even less visible to users.

Reply:
Ok, changed path to `$prefix/.ca_bundle`